### PR TITLE
Support multiple databases

### PR DESCRIPTION
**Pull request by awisema**
This PR adds the ability to pass an AR base class to the fix point test helpers to specify a database to be used.

### CODE REVIEW

File: lib/fixpoint.rb [hunk 1]
```diff
@@ -85,13 +81,13 @@ def fixpoints_path
...
 
-    def read_database_records
+    def read_database_records(conn)
       # adapted from: https://yizeng.me/2017/07/16/generate-rails-test-fixtures-yaml-from-database-dump/
       tables = conn.tables
       tables.reject! { |table_name| TABLES_TO_SKIP.include?(table_name) }
 
       tables.each_with_object({}) do |table_name, acc|
-        result = conn.select_all("SELECT * FROM #{table_name}")
+        result = conn.select_all("SELECT * FROM #{conn.quote_table_name(table_name)}")
```
Comment (motine) [inline lib/fixpoint.rb:90]: good catch to use quote_table_name!.

File: lib/fixpoint_test_helpers.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Helper methods to be included into RSpec
 module FixpointTestHelpers
-  def restore_fixpoint(fixname)
+  def restore_fixpoint(fixname, base: ActiveRecord::Base)
```
Comment (motine) [inline lib/fixpoint_test_helpers.rb]: nice to use a keyword argument here.
why not pass the connection directly? I feel that `base` is a little confusing/generic as a argument name

**Review by motine — CHANGES_REQUESTED** [review]
super cool to add this feature. Thanks so much.

Please add a little documentation in the readme, so people can find you addition.
Also, please reconsider the `base` naming. Other than this, I am really grateful!

File: lib/fixpoint_test_helpers.rb (hunk 2 above)
Comment (awisema) [reply lib/fixpoint_test_helpers.rb]: Done!

File: README.md [hunk 3]
```diff
@@ -54,7 +54,7 @@ RSpec.describe 'User Flow', order: :defined do # !!! mind the order here !!!
     fill_in 'Name', with: 'Tom'
     click_on 'Save'
 
-    store_fixpoint_unless_present :registred_user
```
Comment (motine) [inline README.md:57]: ah cool. thanks!

**Review by motine — APPROVED** [review]
thanks so much!

**Pull request merged**